pkg/adapter/mtping/runner.go

```diff
@@ -107,11 +107,16 @@ func (a *cronJobsRunner) RemoveSchedule(id cron.EntryID) {
 	a.cron.Remove(id)
 }
 
-func (a *cronJobsRunner) Start(stopCh <-chan struct{}) error {
+func (a *cronJobsRunner) Start(stopCh <-chan struct{}) {
 	a.cron.Start()
-	<-stopCh
-	a.cron.Stop()
-	return nil
+	<-stopCh // main channel that gets ready once term signal is received!
```

Comment:
Closed, not "ready". And let's avoid exclamation mark in comments, there is no need to panic :)
```suggestion
	<-stopCh // main channel that gets closed once term signal is received
```